notebooks/uvis_pam_corrections/UVIS Pixel Area Map Corrections.ipynb

```diff
@@ -0,0 +1,955 @@
l2PR9lEJRJiIp/y0wtQrIUY5Q/++CJXHFaae1+zOkBMJ6VGrEqsTdMyu3Q+fUZLX282NMSc6VA2nBRLvqKJayUD0TaTmO+hWUojtWhgtuUAoEF+nz2hmg/dOhJxeJA5oxDtmtI7DbKZe9dE4aU04KE6b9oKhTWKWl2LCVHqLuIiXphlvapsan1id99MYAendANJXqFywMPWqj9o5C7k7TH+PLLZGr/6rv1kR1SyAvxRCXALwEoDPKqX+M4BfBPC0EOIOiCH8jv7+3wFQ1J9/GsA/+GtfRMA0n/oF7vC1CxKdRRPxCheqmxUYzDFfwGnq7MJLDgwdW5/ZCGF3BF2G+lzwkSXQXyDIM7H/9oDC9RDpLYKJStCaLLXPyDovzd43vcWRV+gAwgnhZRXcQoT5chOxJ2p46pEvor8Uor/sYzTvI5h3MZoJkb+uJunRw7LuoXUfXXnIRn8+znl3VsBphShdDhBrKeSus0RvHQemXtPNoG7IfB235xaIg4yzE9w8JzRmHxgUDRj+OL9QoHXUhFBUNPpJAXNATGCcgg1o7QJoWdedlzj0BZbe/Vk5eaC7SxJBTCK1G6J5mhtt85iJYZl4S6Rh6PgO+/lRmdiGOeRmk7sTwnB5+gJA7UE14Q3UWimEuQAiAEZnh0ht8tR3GgrG6P7DnL+hkNpRk5GoiIDmafJaMvcieAVOJ6RGu+qnTTjdCPnbIQZTEvmLpMCP4+2UBKZeY7rZ2Edx/z2hdlAaj3i1Tb3HbA5fm+V0Ximh8LyNIyf20NvKQPYNBDGF+B0H8Q0yiPwsHaABtoS9wwxNUgZP+nEEIFO7DBRu+Ogf0pVayUT9jIHeIYneIQOxdoTs3RBWn6nhdpdanMAhNyV7z6dpsq/QXDWhhEB6iyxOp6WQ2Hfha8Gdl/mbn0pcUko9oJQ6p5Q6o5T6J/rzd5VSjyqlVpRS36+UcvXnR/rvK/rrfzWG+L/68JNM/DV7coIRuEWW+QA1E4VLAvF6gFGRjtFCMcMxsx6iucqg2fzNAF5W31xfYeprY+YXe+sgwRK2c5ghMTIArGGE3oyB1inO1GVA74PmQ7QHVwMTZl/gB973VXzhzJ/glYf+A36xeBsfec/nYLZM/MzbP4upUgd2XaJ2Adh5h4neEZpmFK710TlKf8J4RSGx52qeu0J33qR3Y0ogWaE9/MzzIfqzDiKbzDy3IDRCr/5/6t47SrLrPg/87suvcujc02lyDphBIgEmiUEyKYvKEk1KNC0GSLJkWTTXYb32+njXWit5VwZEBYuKlkRZFCktxSAGECAIDAaYGWBmeqZnOufqyuFVvXj3j+9VjSVKsmcpHwN1Dg7OpO7qW/fd+/v9vjSwjJv8UoQohiwVP542K1TvRTqQvxXArPHhrx/i8LJ8SsPQyz4HbhYPmn5sut6SyC2FcIbpBtQd40Ga2mBP6qWoM0hsKnd5BxqrKasskb8dYOajNyECCXs3rt6KvMW8lAIvSxRICTmTGHs2gt4RSD6fgJ3vIvHNJURlE+VHfPhpgfpDHhK7MmaHUvzmjMaBMhYJaJNPElXYO88SXwQ8UHK3JPyUROmcgt37CWv2D5vA4teDItGYU9E85qO9j4dhYonZlUISSUrsRQNuQ2uGoTqkbgvUH+3B1nykp5qIsgGh1Zj12d0XILUKNI7cPXgSW8rAebw9rg1gRDfH4SIEkFkicczNCUQGiXpuPk5VzysIbJrauBnGITQOsyJ1hrWY5cs21hlREVhMo3LGBVozFvkSyxF051Xo4KQERAj0Fmmso8+Hgw3l5ljyhZbA1ms1WGU6MPeGyLXfeYiT7+acAqvURfZ2hO6QgO6wEqkfRUwWEegNk49g1PlBQAJumlbmqeW+gjAmrmxrMGtAYriDg29ZwtBfctP8cGERt9/9BKb0KtpPjkDtCqRWlYEhRmdCYP3NKeZetElM2TubgJ8i7BfY5Ns7owJ6O0CvKFA9qkF12T/rTR5wtZO8fbNLMTXaJL+/W2BZCcFKp4/7u1m2EFqPqIVUuGnbE9rAPNZP87AUIaHXblFBaxZoTapQXDo3eSk++J1JvtfMWghnVCC5HUAEAq0THje5KVB762F09vGhigwiSe2ZCJVzEkNX4lZBB9whHuLWHglq/lIau5t5nDy7AqFKtOZCmCsmNIeVYO0EW7M+kzSxQ9Oe6lFK760Sh6e6I1F+kwtnTKH4zRPQOqw4+sIorcdWMrHFliqxosMr3s317LsmAUD5DA/k2nEAgv4Jqgu0Z0NEHR03VscRPpuHnvCg9oDxZ3pwxiPkrvHWNuoKuiP8TERAz4/sEu3nVZeHbmtKid3F6Nxlxk7h+Xm2bMXrpKGLEGhNqQM6dt/jw0/zMzabbP0KN0O0pwVe+FdPwM0L5G5FRM1yPHCc0VejVkKSUShVnu61wxrSSxzs9YajgRbArAkkSjRFscp0yMkscSZhlyRW/m4K9SPU5lOvriC9Qt+E5FaE8addFOZDuHkxiFgXUWza0mWV0ZzjTRQaQP28h7liFZ869Bn8ZOGvLnwOGSU4x3vojhESLNyQyCwoyC3yxtXbJPrQ3JTQX2onhFsknTa1KVE6Z8OqENfujJE56Wf49XPzhOZ6eQX1oxK1wwrytwO4BR6Q2UXal/kp/kykWfPf9h2F2OPScVsKDiX9FJC/5cMt0lfQLwZwJu/manQmKRM2mqTUdgvUmrQnaSefvWxAbwHl0yTVZJYQE51o1JpcVzD5JX7t9hTnAHqdIq5ODGwn1wWUjopbT89hZKQBZH24hRDOOLkJmTsMZdG6QP4WWX7N/QLdCbZ3VoWUaC8jMPwFE4mdaJD9aZc4qTcafKCoNaDdvBIA3fEQxRdpmuOOhnDGyV5k28kqwKgTWnWLEu25ENKI8OiZm9DXTM4pXkrBqkVY+g4dI8+TTh2k+sbATCHr8zqopBT4tnc/hdoRxglEGjDyYgA/xRmI2iNLMrAFqscoJutrcKQAUhsesrfuxvgFNlWdeouf
```

Comment:
Insert the correct link for numpy (https://numpy.org/citing-numpy/); the template hasn't been updated yet so that's on me